tests/test_updater_top_level_update.py

```diff
@@ -35,7 +35,7 @@ class TestRefresh(unittest.TestCase):
     def setUp(self) -> None:
         self.temp_dir = tempfile.TemporaryDirectory()
         self.metadata_dir = os.path.join(self.temp_dir.name, "metadata")
-        self.targets_dir = os.path.join(self.temp_dir.name, "targets")
+        self.targets_dir = os.path.join(self.temp_dir.name, Targets.type)
```

Comment:
This is an unrelated "targets" and should not use Targets.type -- in fact it could be renamed "downloads" to be clear about it (it's the directory that client downloads the actual target files to)